.github/workflows/bf-review.yml

```
@@ -87,6 +90,9 @@ jobs:
             dockerfile: ./frontend/benefit/applicant/Dockerfile
             database: false
             port: 3000
+    concurrency:
+      group: ${{ env.K8S_NAMESPACE }}-${{ matrix.service }}
```

Namespace is not available here, but only during the execution from `andersinno/kolga-setup-action@v2`.
```suggestion
      group: ${{ github.event.pull_request.number }}-${{ matrix.service }}
```

---

OK thanks!